src/App.js

```diff
@@ -1,16 +1,55 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+// import ColorChoice from './components/ColorChoice';
 
 const App = () => {
+  const [chatData, setChatData] = useState(chatMessages);
```

Comment:
We need a piece of state to track changes to our messages, initialized from the json file we loaded 💯